src/js/schemas/job-schema/General.js

```diff
@@ -24,26 +26,34 @@ const General = {
         return job.getDescription();
       }
     },
-    cpus: {
-      title: 'CPUs',
-      description: 'The amount of CPUs the job requires',
-      type:'number',
-      getter: function (job) {
-        return `${job.getCpus() || ''}`;
-      }
-    },
-    mem: {
-      title: 'Mem (MiB)',
-      type: 'number',
-      getter: function (job) {
-        return `${job.getMem() || ''}`;
-      }
-    },
-    disk: {
-      title: 'Disk (MiB)',
-      type: 'number',
-      getter: function (job) {
-        return `${job.getDisk() || ''}`;
+    resources: {
+      type: 'group',
+      properties: {
+        cpus: {
+          title: 'CPUs',
+          default: MesosConstants.MIN_CPUS,
```

Comment:
It's so that the form displays the default values when creating a new Job. The getter is only useful for editing, afaik.